src/main/java/core/mate/academy/specificimp/BulldozerProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.specificimp;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private static final int SPECIFIC_LIST = 10;

```

Comment:
weird contant name - how about `NUMBER_OF_BULLDOZERS`?

**should be applied to all other producers**